index

```diff
@@ -291,8 +291,10 @@ bf0025c8b878ed7086c3a29c266ac13be43a8c31a0ea7e5eade717aec00c623d irma-demo/rdw/d
 1a1078dcd7d22e4a30ab34d53a7509256196a470baa883a92c99c073c40690af irma-demo/sidn-pbdf/Issues/email/logo.png
 6629ebbe21ee7cf7e8dff33f7a3dd1109afe4cb071daf2ab31dfafb8e91aacbe irma-demo/sidn-pbdf/Issues/mobilenumber/description.xml
 8aed5653af7f2e2957873f67ec39481f4a9ff6935f738aaa119d2d8bef6db169 irma-demo/sidn-pbdf/Issues/mobilenumber/logo.png
+8255bb29eeaadc0fb17f6772fd9c9c3d68a4322bb2b2c1fad78f61dacea65ba1 irma-demo/sidn-pbdf/Issues/uniqueid/description.xml
+1a1078dcd7d22e4a30ab34d53a7509256196a470baa883a92c99c073c40690af irma-demo/sidn-pbdf/Issues/uniqueid/logo.png
 e71ff28e6327209bf59b44ab17cb3cfb20df50927001c913649eefc48065a6bf irma-demo/sidn-pbdf/PublicKeys/0.xml
-2927a80b538c0bacb20ab3b893d6da40c97f039059fa746b99c5cb8638849821 irma-demo/sidn-pbdf/description.xml
```

Comment:
This file has not been modified, so something has gone wrong while signing. `irma scheme verify` also rejects it when I run it locally.

Reply:
Sloppy of me, thanks for that. I did make some changes and reverted them, but apparently I signed in the middle of that.